test/typescript-jest/basic/app.ts

```diff
@@ -0,0 +1,13 @@
+import { FastifyPlugin } from 'fastify'
+import { join } from 'path'
+const fastifyAutoLoad = require('../../../')
```

Comment:
Could you use import instead?

Reply:
I don't think this is needed, will need install babel-jest and add more settings.